File: src/main/java/com/example/sorokov/model/TokensModel.java
package com.example.sorokov.model;

public class TokensModel {
    private final String accessToken;
    private final String refreshToken;

    public TokensModel(String accessToken, String refreshToken) {
        this.accessToken = accessToken;
        this.refreshToken = refreshToken;
    }

    public String getAccessToken() {
        return accessToken;
    }

    public String getRefreshToken() {
        return refreshToken;
    }

}

File: Models/TokensModel.cs
namespace SorokChatServer.Models
{
    public class TokensModel
    {
        public string AccessToken { get; private set; }
        public string RefreshToken { get; private set; }

        public TokensModel(string accessToken, string refreshToken)
        {
            AccessToken = accessToken;
            RefreshToken = refreshToken;
        }
    }
}
